Detect new brands: Cidea, Cuiud, Daria,  HiBy, mipo and Detect devices for exist brands

sanchezzzhak opened the pull request.

sanchezzzhak committed:
feat(device) detect brand F+: SH65
feat(device) detect brand Oukitel: WP2
feat(device) detect brand Hammer: Energy X
feat(device) detect brand Yandex: YNDX-00092

sanchezzzhak committed:
feat(device) detect new brand W&O: X9 Call 2
feat(device) detect brand OMIX: X400, X600
feat(device) detect brand Nokia: C300, G100
feat(device) detect brand Vortex: HD62
feat(device) detect brand NEXON: X5+
feat(device) detect brand General Mobile: GM 6 DS (G100)

sanchezzzhak committed:
Merge remote-tracking branch 'upstream/master'

sanchezzzhak committed:
Merge remote-tracking branch 'upstream/master'

sanchezzzhak committed:
feat(device) detect new brand Cidea: CM7800 Plus
feat(device) detect new brand Cuiud: F7
feat(device) detect brand ANS: UL40
feat(device) detect brand Kivi: 32H600GU
feat(device) detect brand Lenovo: IdeaTab A1020
feat(device) detect brand Logicom: La Tab Stand Pro
feat(device) detect brand Acer: BattleZone LTE (M10-12_EEA)
feat(device) detect brand Asus: Smartphone for Snapdragon Insiders (I007D)

sanchezzzhak committed:
feat(device) detect brand Blackview: BL8000
feat(device) detect brand ZTE: Blade A53 Pro, 5G UG Phone U23 (7160N), Blade V50 Design 5G (8150N), Blade A73 (7060), Blade A33s, Blade A53+
feat(device) detect brand Alcatel: One Touch Pixi 3 3.5" (A463BG)
feat(device) detect brand Realme: 12 5G (RMX3999)
feat(device) detect brand Xiaomi: Mi 5X
feat(device) detect brand Leagoo: LeaPad 7
feat(device) detect brand Hi: VHIX-40F152MSY
feat(device) detect brand AMCV: LE-50ZTUS30
feat(device) detect brand Sigma: Tab A1010 Neo, X-Treme Tab A1025
feat(device) detect brand Fly: Flylife Connect 10.1 3G 2
feat(device) detect brand Motorola: Moto X40 (XT2301-5)
feat(device) detect brand Vivo: iQOO Z7i (V2230EA)
feat(device) detect brand Dcode: Cygnal 2 Lite (DS-CL2 Lite)

sanchezzzhak committed:
feat(device) detect new brand Daria: DM-B50104
feat(device) detect new brand HiBy: R6 Gen III
feat(device) detect new brand mipo: M25, M46 Pro
feat(device) detect brand Yandex: YNDX-00101
feat(device) detect brand DEXP: 43UCY1
feat(device) detect brand DreamStar: W2
feat(device) detect brand AGM: G2, H6
feat(device) detect brand Nokia: X10
feat(device) detect brand Ulefone: Armor Pad
feat(device) detect brand UMIDIGI: C1 (MP07), F3 (MP09)
feat(device) detect brand LT Mobile: M40 (LT_6509)
feat(device) detect brand Dcode: Cygnal 3 (DS-CL3), Bold 2 (DS-BD2), Cygnal (DS-CL1)
feat(device) detect brand DIJITSU: DCT 90
feat(device) detect brand Mione: Pro+
feat(device) detect brand CCIT: KT200 Pro, KT300 Pro, KT600 Pro
feat(device) detect brand Walton: Nexg N71
feat(device) detect brand TopDevice: A10 (TDT4541_4G_E_RU)
feat(device) detect brand OSCAL: Modern 8, Flat 1C
feat(device) detect brand VGO TEL: New 10, New 20
feat(device) detect brand Sky: Elite A63 Max
feat(device) detect brand Blu: Bold N3
feat(device) detect brand OPPO: K9 Pro 5G, K9s, K9x
feat(device) detect brand Ulefone: Armor X12 Pro, Armor X13, Armor 24, Armor 20WT
feat(device) detect brand Sharp: Aquos R8 (SH-52D)
feat(device) detect brand Rombica: myPhone Jet
feat(device) detect brand Koobee: K100 (SL004T)
feat(device) detect brand BMAX: MaxPad I11 Plus
feat(device) detect brand iHunt: Strong Tablet P15000 Pro
feat(device) detect brand Blackview: BV8900 Pro, BV9300 Pro, BV6200, Mega 1, Active 8, Tab 7 WiFi

liviuconcioiu commented on regexes/device/mobiles.yml:
```diff
@@ -34127,6 +34190,11 @@ Datalogic:
+# mipo (mipo.com.tr)
+mipo:
+  regex: 'mipo[ _]([^/;)]+)(?: Build|[);])'
+  model: '$1'
```
```suggestion
  device: 'smartphone'
  model: '$1'
```

Simbiat commented on Parser/Device/AbstractDeviceParser.php:
```diff
@@ -329,6 +329,7 @@ abstract class AbstractDeviceParser extends AbstractParser
         'C3'  => 'China Mobile',
         'U9'  => 'China Telecom',
         'CI'  => 'Chico Mobile',
+        'CIA' => 'Cidea',
```
Based on photos I've found (for example https://jiji.ng/surulere/tablets/new-c-idea-cm7000-plus-512-gb-rsYzAEHJMsdZor3u7ZfnlZks.html) it should be `C Idea`

sanchezzzhak committed:
Merge remote-tracking branch 'upstream/master' into pr_device_429

sanchezzzhak committed:
feat(device) detect brand Huawei: Nova 11 Ultra (GOA-AL80U)
feat(device) detect brand Ulefone: S11
feat(device) detect brand HIPER: M-Pad
feat(device) detect brand HTC: A100
